next.config.js

```diff
@@ -0,0 +1,19 @@
+const path = require("path");
+const withImages = require("next-images");
+module.exports = withImages({
+  swcMinify: true,
```

Comment:
this creates that .swc folder at root. Speed is incredible.. I can't imagine on a m1